index.html

```diff
@@ -14,22 +14,34 @@
 </head>
 
 <body class="portfolio">
-  <div class="main">
+  <main>
     <nav class="app-bar">
       <a class="My-logo" tabindex="1">My Logo</a>
-      <a class="hamburguer"><img src="img/Enabled.png" alt="sub-menu" class="H-iamge"></a>
+      <button class="hamburguer"></button>
       <ul class="dsk-menu">
         <li class="l-dskmenu">
-          <p tabindex="2">Portfolio</p>
+          <p class="m-menu" tabindex="2">Portfolio</p>
         </li>
         <li class="l-dskmenu">
-          <p tabindex="3">About</p>
+          <p class="m-menu" tabindex="3">About</p>
         </li>
-        <li class="l-dskmenu">
+        <li class="m-menu" class="l-dskmenu">
           <p tabindex="4">Contact</p>
         </li>
       </ul>
     </nav>
+    <ul class="mbl-menu">
+      <li id="close"><button id="btn-close" alt="close menu"><img src="img/x.png" class="img-close"></button></li>
+      <li class="l-dskmenu">
+        <p class="m-menu" tabindex="2">Portfolio</p>
+      </li>
+      <li class="l-dskmenu">
+        <p class="m-menu" tabindex="3">About</p>
+      </li>
+      <li class="m-menu" class="l-dskmenu">
+        <p tabindex="4">Contact</p>
+      </li>
+    </ul>
```

Comment:
- [ ] As specified in the project requirements please make sure that when the user clicks any of the mobile menu options, the correct part of the page is displayed. So that means the above paragraph elements should be a link tag that should redirect to the corresponding section of the page. ✔
- Here's the project requirement as a reminder:
  ![image](https://user-images.githubusercontent.com/104968236/232763938-3fca98ed-af1c-439d-a112-5fc2c5159f17.png)